### DISCUSSION TITLE
Multi layer with names

### DISCUSSION THREAD

**Discussion by Naveena1321** [clientIO/joint#2778] [Q&A] [answered]
### Introduction

Do we have anything in jointjs supporting creation of multiple layers one have larger zindex than other so that on clicking send to back only moving that layer back

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment, marked as answer]
This needs to be implemented at the logical level.

Currently, stacking order is managed only by the `z` attribute, and all cells are added to a single layer, `dia.Paper.Layers.CELLS`.

Adding the API to manage layers and to specify a layer for each cell is on our roadmap.
